src/site/content/en/accessible/color-and-contrast-accessibility/index.md

```diff
@@ -54,6 +60,24 @@ You will get a detailed report of any failing elements.
   <img src="./fastpass-contrast.jpg" alt="The report in Accessibility Insights">
 </figure>
 
+## Advanced Perceptual Contrast Algorithm (APCA)
+
+The [Advanced Perceptual Contrast Algorithm (APCA)](https://w3c.github.io/silver/guidelines/methods/Method-font-characteristic-contrast.html) is a new way to compute contrast based on modern research on color perception.
```

Comment:
I did not explain about the Silver Task Force here: https://w3c.github.io/silver/. Let me know if you think it's good to do so. @mathiasbynens